app/viewmodels/SummaryCardHelper.scala

```diff
@@ -322,9 +324,10 @@ class SummaryCardHelper @Inject()(link: views.html.components.link) extends Impl
 
       (penaltyAppealStatus, periodSubmissionStatus, penaltyPointStatus) match {
         case (Some(AppealStatusEnum.Accepted | AppealStatusEnum.Accepted_By_Tribunal), _, _) => renderTag(messages("status.cancelled"))
-        case (_, Some(Submitted), Due) => renderTag(messages("status.due"), "penalty-due-tag")
+//        case (_, Some(Submitted), Due) => renderTag(messages("status.due"), "penalty-due-tag")
```

Comment:
Can the commented lines removed, or do we want to keep it.

Reply:
Can be removed, will clean this up.